cli/certs_test.go

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package cli_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/absmach/magistrala/cli"
+	"github.com/absmach/magistrala/pkg/errors"
+	svcerr "github.com/absmach/magistrala/pkg/errors/service"
+	mgsdk "github.com/absmach/magistrala/pkg/sdk/go"
+	sdkmocks "github.com/absmach/magistrala/pkg/sdk/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+const (
+	revokeCommand = "revoke"
+	issueCommand  = "issue"
+)
```

Comment:
Move all command constants to the `commands_test.go` file. Keep them in separate `const` blocks and put comments explaining to which commands they refer:
```go

// Certs commands
const (
    revokeCmd = "revoke"
    issueCmd = "issue"
)
```
Do the same for the other services. Also, group common commands, something like:
```go

// CRUD and common commands
const (
    createCmd = "create"
    updateCmd = "update"
    getCmd = "get"
    enableCmd = "enable"
    ...
)
```